### PR TITLE
Add docs for Kaptain  1.3.0 release

### PR DESCRIPTION
**Pull request by timcharper**
http://docs-d2iq-com-pr-4050.s3-website-us-west-2.amazonaws.com/

## Jira Ticket

https://jira.d2iq.com/browse/D2IQ-83703

## Description of changes being made

Sync of documentation from Kaptain repo (release/kaptain-1.3 branch), as of commit b4c27dea016b60fb548b9795b5b4287ec16c01ba from the time of this writing.

A lot of this documentation is copied from the 1.2.0 release. At a glance of the changes since tag `kaptain-sdk-0.4.0`, these pages have more substantial changes and should be the focus:

* release-notes/1.3.0/index.md
* install/air-gapped/index.md
* install/konvoy-dkp/index.md
* tutorials/sdk/quick-start/index.md
* tutorials/sdk/image-builder/index.md

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Create new PRs against `develop`, or `main` for hotfixes to production.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

File: pages/dkp/kaptain/1.3.0/release-notes/1.3.0/index.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+layout: layout.pug
+navigationTitle:  Release Notes for Kaptain 1.3.0
+title: Release Notes for Kaptain 1.3.0
+menuWeight: 5
+beta: false
+excerpt: View release-specific information for Kaptain
+---
+
+**D2iQ&reg; Kaptain&reg; version 1.3 was released on January 14, 2022.**
```
Comment (cmastr) [inline pages/dkp/kaptain/1.3.0/release-notes/1.3.0/index.md:10]:
```suggestion
**D2iQ&reg; Kaptain&reg; version 1.3 was released on January 19, 2022.**
```

File: pages/dkp/kaptain/1.3.0/release-notes/1.3.0/index.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+### Improvements
+#### Kaptain support on DKP 2.1
+
+Kaptain 1.3 is supported on version 2.1 of DKP, D2iQ's enterprise Kubernetes platform, in both non-air-gapped and air-gapped environments. For more information on DKP, view Konvoy and Kommander documentation on [docs.d2iq.com](https://docs.d2iq.com/).
+
+In DKP 2.1, Kaptain's dependencies are managed through Kommander, the centralized management plane of DKP which handles application deployments.  
+
+#### Installation and Upgrade Improvements
+
+The Kaptain installation and upgrade time has been shortened in 1.4.
```
Comment (chessahoekstra) [inline pages/dkp/kaptain/1.3.0/release-notes/1.3.0/index.md:48]:
```suggestion
The Kaptain installation and upgrade time has been shortened in 1.3.
```

Comment (timcharper) [reply pages/dkp/kaptain/1.3.0/release-notes/1.3.0/index.md:48]: Good catch!

**Review by cneth — APPROVED** [review]

**Review by cmastr — APPROVED** [review]

File: pages/dkp/kaptain/1.3.0/introducing/index.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+If you want to learn more, please read our [blog post for Kaptain](https://d2iq.com/blog/kudo-for-kubeflow-the-enterprise-machine-learning-platform).
+
+## Kaptain's Features and Benefits
+
+| **Features**                                      | **Benefits**                                                 |
+| ------------------------------------------------- | ------------------------------------------------------------ |
+| Out-of-the-box integration of Spark and Horovod   | No need to install additional libraries to create data pipelines or train Spark ML models on multiple CPUs or GPUs |
+| Fully tested pre-baked notebook images            | A familiar environment that has been fully tested and integrates with all the shared resources (CPUs, GPUs) and data access controls needed to build and share models as a team |
+ |
```
Comment (msdolbey) [inline pages/dkp/kaptain/1.3.0/introducing/index.md:21]:
```suggestion
```

File: pages/dkp/kaptain/1.3.0/tutorials/sdk/tensorflow/index.md [hunk 4]
```diff
@@ -0,0 +1,525 @@
...
+
+## How to Create a Docker Credentials File and Kubernetes Secret
+
+For the tutorial you will need `getpass` to provide a password interactively without it being immediately visible.
+It is a standard Python library, so there is no need to install it.
+A simple `import` will suffice.
+
+<p class="message--warning"><strong>WARNING: </strong>Please do not store passwords directly in notebooks.
+    Ideally, credentials are stored safely inside secrets management solutions or provided with service accounts.
+    Please check the section on <a href="https://docs.d2iq.com/dkp/kaptain/">how to manage secrets</a> in the official Kaptain documentation for more details on how to set up Docker credentials that you can attach to a notebook server.
```
Comment (msdolbey) [inline pages/dkp/kaptain/1.3.0/tutorials/sdk/tensorflow/index.md:79]:
```suggestion
    Please check the section on <a href="https://docs.d2iq.com/dkp/kaptain/1.3.0/secrets-management/">how to manage secrets</a> in the official Kaptain documentation for more details on how to set up Docker credentials that you can attach to a notebook server.
```

**Review by msdolbey — APPROVED** [review]
👍 
lovely

**Review by Colin1841 — APPROVED** [review]

**Comment by msdolbey** [comment]
So, there are approx 1 zillion Vale errors, I feel like that can be taken on after this gets merged. That might take too long to untangle for a single PR.

**Pull request merged**